mirage/scenarios/registrations.ts

```diff
@@ -128,13 +137,13 @@ export function registrationScenario(
 
     const decaf = server.create('registration', {
         id: 'decaf',
-        title: 'Pending Penguins',
+        title: 'Global Penguins',
         registrationSchema: server.schema.registrationSchemas.find('testSchema'),
         provider: egap,
         reviewsState: RegistrationReviewStates.Accepted,
         registeredBy: currentUser,
         revisionState: RevisionReviewStates.RevisionPendingModeration,
-        currentUserPermissions: Object.values(Permission),
+        currentUserPermissions: [Permission.Admin],
```

Comment:
This is not an appropriate change. `Permission.Admin` is included as part of the permissions from `Object.values(Permission)` and having the other permissions included in the list is more correct.

Reply:
Ok, this was done for testing purposes and to render the page. If not helpful, this is being changed to its original value.

Reply:
This is showing as updated so marking this as resolved.